.pre-commit-config.yaml

```
@@ -1,7 +1,7 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 default_language_version:
-  python: python3
+  python: python3.11
```

I think we should avoid specifying the version here so it's one less place to keep track of

```suggestion
  python: python3
```